server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -2437,6 +2442,16 @@ public void importMapFiles(long tid, Map<FileRef,MapFileInfo> fileMap, boolean s
           }
         }
       }
+
+      Iterator<FileRef> fiter = fileMap.keySet().iterator();
```

Comment:
I bet there's a removeIf construct in Java 8 you can use for the master branch.

Reply:
When I first wrote this code I used multiple Java 8 map methods.  Then I realized I couldn't.

This particular code could have been implement using `fileMap.keySet().removeAll(bulkImporting)`.  However then there would be no logging.  I thought the logging was important, so wrote it this way which was the most efficient way I could think of writing it.